manifests/service.pp

```diff
@@ -4,4 +4,12 @@
     enable   => $filebeat::service_enable,
     provider => $filebeat::service_provider,
   }
+  
+  if $filebeat::service_enable and $::kernel == 'FreeBSD' {
+    file_line {'/etc/rc.conf: filebeat_enable="YES"':
+      ensure => present,
+      path   => '/etc/rc.conf',
+      line   => 'filebeat_enable="YES"',
+    }
+  }
```

Comment:
I think that the service enable piece is handled automatically by the BSD `service` resource provider. If true, we can leave this piece out.